support-frontend/assets/helpers/subscriptionsForms/validation.ts

```diff
@@ -36,11 +30,11 @@ function nonSillyCharacters(s: string | null | undefined): boolean {
 function firstError<FieldType>(
 	field: FieldType,
 	errors: Array<FormError<FieldType>>,
-): Option<ErrorMessage> {
-	const msgs = errors
+): string | undefined {
+	const msgs: string[] = errors
 		.filter((err) => err.field === field)
 		.map((err) => err.message);
-	return headOption(msgs);
+	return msgs[0];
```

Comment:
I didn't realise we had all these scala style types kicking about

Reply:
Yup they are from way back, I think we should ruthlessly remove them now though.